mdal/frmts/mdal_ugrid.cpp

```diff
@@ -782,10 +999,6 @@ void MDAL::DriverUgrid::writeVariables( MDAL::Mesh *mesh )
     }
   }
 
-  // Time array
-  int timeId = mNcFile->defineVar( "time", NC_DOUBLE, 1, &dimTimeId );
-  mNcFile->putAttrStr( timeId, "units", "hours since 2000-01-01 00:00:00" );
-
```

Comment:
why this is removed?

Reply:
This method is called only for writing mesh frame. In that case, there is no time, and I think UGRID supports no time. Further more, settings a dummy time here leads to not be able to check if the time is the same when writing new dataset group.
The logic is:
- if there is already times in the file, MDAL checks if the times (ref and time steps) are the same, error if not
- if there is no times, adds time depending of the times of the new dataset group.